### ISSUE TITLE
KB config needs to be fixed.

### ISSUE THREAD

**Issue by Aleksey-Terzi** [CivMC/Finale#24]
The current version of Finale config has an unsupported description of knockback modifiers:
```
  sprintMultiplier: # modifies knockback if the attacker is sprinting.
    x: 2.3
    y: 1.0
    z: 2.3
  knockbackMultiplier: # modifies knockback in general.
    x: 1.2
    y: 0.9
    z: 1.2
  waterKnockbackMultiplier: # modifies knockback if the victim is in water.
    x: 1.0
    y: 1.0
    z: 1.0
```
Though the format supported by Finale is like this:
```
  knockbackLevelMultiplier: 0.6 # a constant to control the knockback sword level
  normal:
    groundModifier: # modifies knockback in general.
      type: DIRECT # MULTIPLY or ADD or DIRECT
      x: 0.625
      y: 0.5
      z: 0.625
    airModifier: # modifies knockback if the victim is in the air.
      type: DIRECT # MULTIPLY or ADD or DIRECT
      x: 0.625
      y: 0.5
      z: 0.625
    waterModifier: # modifies knockback if the victim is in water.
      type: DIRECT # MULTIPLY or ADD or DIRECT
      x: 0.625
      y: 0.5
      z: 0.625
  sprint: # modifies knockback if the attacker is sprinting.
    groundModifier: # modifies knockback in general.
      type: DIRECT # MULTIPLY or ADD or DIRECT
      x: 1.0
      y: 0.6
      z: 1.0
```

Draw attention that knockback modifiers don't work now, this PR needs to be applied to make them workable: https://github.com/CivMC/Finale/pull/23

**Comment by Aleksey-Terzi** [comment]
Modifiers from the default Finale config are similar to vanilla modifiers

**Comment by Diet-Cola** [comment]
Fixed by #23
